### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by dtranquang**

### CODE REVIEW

**Comment by laurentleseigneur** [comment]
LGTM

File: md/release-notes.md [hunk 1]
```diff
@@ -107,3 +107,60 @@ The most wanted choice would be to re-create the custom layout from Bonita 7.5.0
...
+* BS-16241 X-Bonita-API-Token Duplicated in login response header
+* BS-16273 SVN repository Migration loses Form critical files
+* BS-16280 StringIndexes / search keys are not updated in sub-processes
+* BS-16282 Parent process stringindex (search key) is updated with a human task's operation part of a subprocess
+* BS-16288 Cannot use Japaneses in Repository name
+* BS-16291 To delete and then instantiate a BO list in the same task operations fails
+* BS-16292 Create a new form is missing
+* BS-16298 TimeoutException is hidden when it occurs during actor filter execution
+* BS-16311 BDM deployment confirm message not translated in French
+* BS-16317 ClosedChannelException occured when bitronix is writing to its DiskJournal
```
Comment (danila-m) [inline md/release-notes.md]: This one is not "fixed" in 7.5.0. We have a solution but it's a patch.

**Comment by danila-m** [comment]
LGTM otherwise.

File: md/release-notes.md (hunk 1 above)
Comment (educhastenier) [reply md/release-notes.md]: Removed it

**Pull request merged**